### DISCUSSION TITLE
Just converted my Neovim config to use norg and tangle, and it works really well.

### DISCUSSION THREAD

**Discussion by simonhughxyz** [nvim-neorg/neorg#1341] [General]
Just recently, as an experiment, I decided to convert and write my Neovim config in norg and use tangle.
To my surprise, it works much better then I thought it would. I thought I it would constantly break, particularly with a complex config full of code blocks but so far it has been rock solid.

So this is a bit of a well done and thank you.

If anyone is interested, my config is [here](https://github.com/simonhughxyz/NeovimConfig)

**Comment by noctopixi** [comment]
Hello! Thanks for sharing your config. I see that you use Looking Glass. Have you been able to get the LSP to work inside Looking Glass buffers?
In my case, the LSP does not kick in, I get no completions, line numbers are off and code opens folded by default. I'm really at a loss on how to change this.

**Reply by simonhughxyz** [reply]
No LSP does not work with looking glass.
There is an issue for this #1123 and a possible fix, but no PR has been made as far as I can tell.
There is a PR to allow LSP inside the code blocks #1329 but I don't think it will fix the looking glass lsp though.

**Reply by noctopixi** [reply]
Hello! Looks like the PR you mentioned was merged recently and only requires an extra plugin, I will try it out.
Thank you so much for pointing it out!
